File: python/max_consecutive_ones.py
from typing import List


class Solution:
    def findMaxConsecutiveOnes(self, array: List[int]) -> int:
        result, current = 0, 0
        for element in array:
            if element is 1:
                current += 1
                result = max(result, current)
            else:
                current = 0
        return result

File: java/src/MaxConsecutiveOnes.java
public class MaxConsecutiveOnes {
    public int findMaxConsecutiveOnes(int[] array) {
        int result = 0, current = 0;
        for (int element : array) {
            if (element == 1) {
                current++;
                result = Math.max(result, current);
            } else {
                current = 0;
            }
        }
        return result;
    }
}
